applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_transonic_perturbation_potential_element.cpp

```diff
@@ -245,14 +235,13 @@ KRATOS_TEST_CASE_IN_SUITE(TransonicPerturbationPotentialFlowSupersonicDecelerati
     pElement->Initialize(model_part.GetProcessInfo());
     pUpwindElement->SetFlags(INLET);
 
-    // velocity[0] : 394.18
-    // velocity[1] : -50
-    // local_mach_number_squared : 1.705
-    AssignPotentialsToSupersonicTransonicPerturbationElement(pElement);
+    std::array<double, 3> high_potential{10.0, 200.0, 150.0};
```

Comment:
```suggestion
    std::array<double, 3> high_potential{10.0, 200.0, 350.0};
```

Reply:
I'm a bit confused why to changed these?

Reply:
As far as I understand the current element has nodes 1, 2 and 3 and the upwind element has nodes 1, 3 and 4, so they share nodes 1 and 3 and in these nodes the potential value should be the same.

Reply:
Oh yes!